darts/tests/models/forecasting/test_global_forecasting_models.py

```diff
@@ -164,6 +173,45 @@ def test_save_model_parameters(self):
                     model._model_params, model.untrained_model()._model_params
                 )
 
+        def test_save_load_model(self):
+            # check if save and load methods work and if loaded model creates same forecasts as original model
+            cwd = os.getcwd()
+            os.chdir(self.temp_work_dir)
+
+            for model in [
+                RNNModel(
+                    input_chunk_length=4, hidden_dim=10, batch_size=32, n_epochs=10
+                ),
+                TCNModel(
+                    input_chunk_length=4,
+                    output_chunk_length=3,
+                    n_epochs=10,
+                    batch_size=32,
+                ),
+            ]:
+                forecasting_horizon = 4
+                model_path = type(model).__name__
+
+                full_model_path = os.path.join(self.temp_work_dir, model_path)
+
+                model.fit(self.ts_pass_train)
+                model_prediction = model.predict(forecasting_horizon)
+
+                # test save
+                model.save()
```

Comment:
we might want to check here as well that the file is saved at the desired location when not giving a path.
You can see everything that was saved with `os.walk(self.temp_work_dir)`.